oaknational/Oak-Web-Application
# Exclude "all" from subject categories in english primary/secondary

## Description
This PR adds to our "fake exclusion table" exclusions for subject categories

```js
{
  subjectcategories: {
    all_disabled: true,
    default_category_id: 19,
  },
}
```

## Issue(s)

Fixes `CUR-830`

## How to test
Cycle 2 (**not enabled**)
1. Go to https://deploy-preview-2950--oak-web-application.netlify.thenational.academy
2. Navigate to the curriculum visualiser english
3. Nothing should have changed, no regressions

Cycle 2 (**is enabled**)

1. Go to https://deploy-preview-2950--oak-web-application.netlify.thenational.academy
2. Navigate to the curriculum visualiser english
3. The "All" subject category should no longer be present

### Screenshots
Before

<img width="880" alt="Screenshot 2024-10-31 at 16 42 52" src="https://github.com/user-attachments/assets/76af6dd3-2d9a-43a5-8150-bfefa5263c0b">

After

<img width="880" alt="Screenshot 2024-10-31 at 16 42 59" src="https://github.com/user-attachments/assets/5b956a56-d76e-4852-b042-0c72ff3367b5">
